### PR TITLE
 Harmonized frame level data structure between encoder and decoder 

### PR DESCRIPTION
**Pull request by nijilk**
Verified that encoder is generating same bit-stream after the changes for limited set of sequences.

### CODE REVIEW

**Comment by joelsole** [comment]
This looks good to be merged. Please, could you rebase to resolve conflicts and check for warnings? Thanks!

**Comment by nijilk** [comment]
> 
> 
> This looks good to be merged. Please, could you rebase to resolve conflicts and check for warnings? Thanks!

@joelsole 
Thanks for the review. I will fix this as soon as PR #429 is accepted.

**Comment by joelsole** [comment]
LGTM. 
@hassount ?

File: Source/Lib/Common/Codec/EbAv1Structs.h [hunk 1]
```diff
@@ -6,10 +6,19 @@
...
+#include "EbSegmentationParams.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/* Maximum number of tile rows and tile columns
+  Conflicting with wrong values entered in EbDefinitions */
+
+#define MAX_TILE_ROWS_AV1 64
```
Comment (hassount) [inline Source/Lib/Common/Codec/EbAv1Structs.h]: this was addressed in #436
could we harmonize the marcos

**Comment by hassount** [comment]
LGTM apart from the TIle count macros which once addressed we can move forward

**Comment by hassount** [comment]
This PR is next on the list, can it be rebased so that we can merge it ?

**Comment by nijilk** [comment]
> 
> 
> This PR is next on the list, can it be rebased so that we can merge it ?

@hassount 
Sure. Will do immediately.

**Pull request merged**